tests/test_integration.py

```diff
@@ -150,6 +164,29 @@ def test_worker_service_called_multiple_times(self):
         eq_(payload['backend'], {'app': 'django', 'app_type': 'web'})
         eq_(payload['database'], {'app': 'postgres', 'app_type': 'db'})
 
+    def test_worker_http_error_logging(self):
+        # Tests the logging http error logic
+        tracer = self.tracer
+        self.tracer.writer.api = Flawed_API('localhost', 7777)
+        tracer.trace('client.testing').finish()
+        tracer.trace('client.testing').finish()
+        eq_(tracer.writer._worker._last_error_ts, 0)
+
+        with LogCapture(level=logging.ERROR) as l:
```

Comment:
just use a mocked logging handler instead of the `testfixtures` package:
http://stackoverflow.com/questions/899067/how-should-i-verify-a-log-message-when-testing-python-code-under-nose